lib/Doctrine/DBAL/Driver/PDOConnection.php

```diff
@@ -104,11 +104,13 @@ public function quote($value, $type = ParameterType::STRING)
     public function lastInsertId($name = null)
     {
         try {
-            if ($name === null) {
```

Comment:
Let's keep it with an if for more accurate code coverage.

Reply:
Done.